receiver/sqlserverreceiver/obfuscate_test.go

```diff
@@ -0,0 +1,256 @@
osoft.com/sqlserver/2004/07/showplan" ScalarString="DM_XE_SESSION_TARGETS. [ target_name ] = N ?"><Compare xmlns="http://schemas.microsoft.com/sqlserver/2004/07/showplan" CompareOp="EQ"><ScalarOperator xmlns="http://schemas.microsoft.com/sqlserver/2004/07/showplan"><Identifier xmlns="http://schemas.microsoft.com/sqlserver/2004/07/showplan"><ColumnReference xmlns="http://schemas.microsoft.com/sqlserver/2004/07/showplan" Table="[DM_XE_SESSION_TARGETS]" Column="target_name"></ColumnReference></Identifier></ScalarOperator><ScalarOperator xmlns="http://schemas.microsoft.com/sqlserver/2004/07/showplan"><Const xmlns="http://schemas.microsoft.com/sqlserver/2004/07/showplan" ConstValue="N ?"></Const></ScalarOperator></Compare></ScalarOperator></Predicate></Filter></RelOp><RelOp xmlns="http://schemas.microsoft.com/sqlserver/2004/07/showplan" NodeId="3" PhysicalOp="Table-valued function" LogicalOp="Table-valued function" EstimateRows="1" EstimateIO="0" EstimateCPU="1.157e-06" AvgRowSize="9" EstimatedTotalSubtreeCost="3.17798e-05" Parallel="0" EstimateRebinds="25.9994" EstimateRewinds="4.62341" EstimatedExecutionMode="Row"><OutputList xmlns="http://schemas.microsoft.com/sqlserver/2004/07/showplan"></OutputList><TableValuedFunction xmlns="http://schemas.microsoft.com/sqlserver/2004/07/showplan"><DefinedValues xmlns="http://schemas.microsoft.com/sqlserver/2004/07/showplan"></DefinedValues><Object xmlns="http://schemas.microsoft.com/sqlserver/2004/07/showplan" Table="[DM_XE_SESSIONS]"></Object><ParameterList xmlns="http://schemas.microsoft.com/sqlserver/2004/07/showplan"><ScalarOperator xmlns="http://schemas.microsoft.com/sqlserver/2004/07/showplan" ScalarString="( ? )"><Const xmlns="http://schemas.microsoft.com/sqlserver/2004/07/showplan" ConstValue="( ? )"></Const></ScalarOperator><ScalarOperator xmlns="http://schemas.microsoft.com/sqlserver/2004/07/showplan" ScalarString="( ? )"><Const xmlns="http://schemas.microsoft.com/sqlserver/2004/07/showplan" ConstValue="( ? )"></Const></ScalarOperator><ScalarOperator xmlns="http://schemas.microsoft.com/sqlserver/2004/07/showplan" ScalarString="DM_XE_SESSION_TARGETS. [ event_session_address ]"><Identifier xmlns="http://schemas.microsoft.com/sqlserver/2004/07/showplan"><ColumnReference xmlns="http://schemas.microsoft.com/sqlserver/2004/07/showplan" Table="[DM_XE_SESSION_TARGETS]" Column="event_session_address"></ColumnReference></Identifier></ScalarOperator><ScalarOperator xmlns="http://schemas.microsoft.com/sqlserver/2004/07/showplan" ScalarString="( ? )"><Const xmlns="http://schemas.microsoft.com/sqlserver/2004/07/showplan" ConstValue="( ? )"></Const></ScalarOperator><ScalarOperator xmlns="http://schemas.microsoft.com/sqlserver/2004/07/showplan" ScalarString="N ?"><Const xmlns="http://schemas.microsoft.com/sqlserver/2004/07/showplan" ConstValue="N ?"></Const></ScalarOperator></ParameterList></TableValuedFunction></RelOp></NestedLoops></RelOp></QueryPlan></StmtSimple></Statements></Batch></BatchSequence></ShowPlanXML>`
```

Comment:
Same here, move this to be own file that is loaded into the test.

Reply:
fixed in https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/37958/commits/92ecd54fa275ac3c290ac3cb590e499969cd903e